llvm/lib/CodeGen/SelectionDAG/SelectionDAGISel.cpp

```diff
@@ -180,6 +182,15 @@ static const bool ViewDAGCombine1 = false, ViewLegalizeTypesDAGs = false,
                   ViewSchedDAGs = false, ViewSUnitDAGs = false;
 #endif
 
+#define ISEL_DUMP(X)                                                           \
+  do {                                                                         \
```

Comment:
Do we need to define this to an empty block when NDEBUG is defined? Otherwise I think we try to evaluate `X` but the `FuncName` variable passed to the first ISEL_DUMP won't be defined.

Reply:
Oops, I thought I passed `X` into `LLVM_DEBUG` (which will not evaluate X in NDEBUG). It's fixed now.